util/schemachecker/schema_checker.go

```diff
@@ -0,0 +1,66 @@
+// Copyright 2018 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package schemachecker
+
+import (
+	"sync/atomic"
+	"time"
+
+	"github.com/pingcap/tidb/domain"
+	"github.com/pingcap/tidb/metrics"
+)
+
+// SchemaLeaseChecker is used for checking schema-validity.
+type SchemaLeaseChecker struct {
+	domain.SchemaValidator
+	schemaVer       int64
+	relatedTableIDs []int64
+}
+
+var (
+	// SchemaOutOfDateRetryInterval is the sleeping time when we fail to try.
```

Comment:
s/sleeping time/backoff time/

Reply:
"fail to try" or "try"

Reply:
It's original comments.
Do you have a suggestion?